### DISCUSSION TITLE
The list krdns filter has not been updated since April 6th and 7th

### DISCUSSION THREAD

**Discussion by sssh7710** [List-KR/community#9] [Q&A]
![Screenshot_20230408_093206_AdGuard](https://user-images.githubusercontent.com/98168192/230695785-32ff6771-042d-406e-861f-fa9328ee04eb.png)

![Screenshot_20230408_093812_AdGuard](https://user-images.githubusercontent.com/98168192/230695795-ce723e13-afbe-419e-9480-d827e20e8fae.png)

Is there a problem?

**Comment by sssh7710** [comment]
Is it right to put it up here?

**Comment by sssh7710** [comment]
I mean, dns filter update error

**Reply by piquark6046** [reply]
Did you try to register it with the following URL?
`https://cdn.jsdelivr.net/gh/adguardteam/HostlistsRegistry@main/assets/filter_25.txt`
